src/page/SignInStep.page.ts

```diff
@@ -0,0 +1,21 @@
+import { ElementFinder, promise, element, by } from 'protractor';
+
+export class SignInStepPage {
+  private get signInButton(): ElementFinder {
+    return element(by.css('#SubmitLogin > span'));
+  }
+
+  public sendMail(inputEmailAddress): promise.Promise<void> {
+    return element(by
```

Comment:
get the element in other method first, that will help in the case you need to check if the field is filled properly (not in this workshop, but in other projects is required)